### ISSUE TITLE
Where is Batch Norm decay 0.997 set?

### ISSUE THREAD

**Issue by KaqiKaqi** [google/automl#239]
Hi, in the paper Batch Norm decay is set to 0.997. But in hyperparameters, I can't find 0.997. Where to set Batch Norm decay?

**Comment by fsx950223** [comment]
> Hi, in the paper Batch Norm decay is set to 0.997. But in hyperparameters, I can't find 0.997. Where to set Batch Norm decay?

It's 0.998 now.

**Comment by mingxingtan** [comment]
@KaqiKaqi , the old value is 0.997, but then I realized that I should use the same bn as backbone (0.99), so I  changed them later for both code and arxiv paper.
